src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -117,6 +117,10 @@ Copyright (c) .NET Foundation. All rights reserved.
       <LangVersion Condition="'$(LangVersion)' == ''">$(MaxSupportedLangVersion)</LangVersion>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <RazorCompilationErrorLog Condition="'$(RazorCompilationErrorLog)'==''">$(IntermediateOutputPath)$(RazorTargetName).Compilation.log</RazorCompilationErrorLog>
```

Comment:
Remind me what `RazorTargetName` is going to be. Isn't it already distinct from `TargetName` so we don't need `.Compilation` at the end.

Reply:
I think we just need to specify a different property to pass to the `Csc` task not, actually define it. `ErrorLog` isn't defined by default, and my guess is that we expect users to specify a value for it if they're interested in the diagnostics for that specific compilation. @nguerrera do correct me if I'm wrong.

Reply:
It's a bit moot if we go with what @pranavkm says, but the answer to your question is basically `$(AssemblyName).[Razor|Views]`.

Reply:
Yeah that's probably a good point. We don't need to give these values, we just need to make it possible to define. 👍